demos/youtube/src/main/java/com/guru/composecookbook/youtube/components/YoutubeListItem.kt

```diff
@@ -37,7 +37,7 @@ fun YoutubeListItem(item: Tweet) {
             .clickable { }
     ) {
         val (image, authorImage, title, subtitle, button) = createRefs()
-        createVerticalChain(title, subtitle, chainStyle = ChainStyle.Packed)
```

Comment:
has this been deprecated ?

Reply:
it is not, when i clicked it there is no deprecated annotation and the text is not strikethrough too. But it has a TODO that say it needs improvement

Reply:
Gotcha, Thanks for the fix :)